common/config/rush/pnpm-lock.yaml

```diff
@@ -1,10 +1,19 @@
 dependencies:
+  '@azure/abort-controller': 1.0.0-preview.2
   '@azure/amqp-common': 1.0.0-preview.6_rhea-promise@0.1.15
   '@azure/arm-servicebus': 3.2.0
+  '@azure/core-arm': 1.0.0-preview.3
   '@azure/core-asynciterator-polyfill': 1.0.0-preview.1
+  '@azure/core-auth': 1.0.0-preview.3
+  '@azure/core-http': 1.0.0-preview.3
+  '@azure/core-paging': 1.0.0-preview.2
+  '@azure/core-tracing': 1.0.0-preview.2
   '@azure/cosmos-sign': 1.0.2
   '@azure/eslint-plugin-azure-sdk': 2.0.1_9e8391ca70fb0408a9da4393803aa477
   '@azure/event-hubs': 2.1.1
+  '@azure/identity': 1.0.0-preview.3
+  '@azure/keyvault-keys': 4.0.0-preview.5
+  '@azure/keyvault-secrets': 4.0.0-preview.5
```

Comment:
Should `@azure/keyvault-certificates` be here as well?

Reply:
This section is for packages that appear under the `dependencies` section of other packages.
We have keyvault-keys being a dev dependency in identity and keyvault-secrets a dependency in the smoke test package